src/openloco/objects/interface_skin_object.h

```diff
@@ -161,17 +161,17 @@ namespace openloco
 
         constexpr uint32_t toolbar_menu_industries = 191;
 
-        constexpr uint32_t build_vehicle_train = 251;
+        constexpr uint32_t build_vehicle_train = 251; // 15 frames
```

Comment:
Fwiw, we've so far been using image ids for all frames involved. See e.g. `ScenarioOptions::drawTabs`.